Automattic/newspack-theme
# Update article post meta

### All Submissions:


### Changes proposed in this Pull Request:

This PR tidies up the post meta for each post:

1. It makes it consistent on the single posts and blog posts page (displays the author and date information at the top, and the category, tag and edit link at the bottom).
2. Removes the comments link from the top for now.
3. Adds the author avatar, and aligns the post author's name and date next to it. 
4. Removes the icons from the post author and date. 
5. Updates the post header template part to be used on single and archive pages. 
6. Removes the 'Featured' text from sticky posts, and the associated styles.


### How to test the changes in this Pull Request:

1. Apply the PR.
2. Confirm that only the author avatar, name and date appear at the top of posts, without icons. The author name and date should sit next to the author avatar.
3. Confirm that the categories, tags and edit link appear at the bottom of posts.
4. Create a sticky post (or check an existing sticky post) on the block posts page and confirm that it does not say 'Featured' at the top. 

### Other information:
